packages/typespec-ts/src/transform/transform.ts

```diff
@@ -38,29 +35,18 @@ import { transformRLCOptions } from "./transfromRLCOptions.js";
 import { transformApiVersionInfo } from "./transformApiVersionInfo.js";
 import { getClientLroOverload } from "../utils/operationUtil.js";
 import { transformTelemetryInfo } from "./transformTelemetryInfo.js";
-
-export interface RLCSdkContext extends SdkContext {
-  options?: RLCOptions;
-}
+import { SdkContext } from "../utils/interfaces.js";
 
 export async function transformRLCModel(
   program: Program,
   emitterOptions: RLCOptions,
   client: SdkClient,
-  emitterOutputDir: string,
-  dpgContext: RLCSdkContext
+  dpgContext: SdkContext
 ): Promise<RLCModel> {
-  const options: RLCOptions = transformRLCOptions(
-    emitterOptions,
-    emitterOutputDir,
-    dpgContext
-  );
-  dpgContext.options = options;
+  const options: RLCOptions = transformRLCOptions(emitterOptions, dpgContext);
+  dpgContext.rlcOptions = options;
```

Comment:
```suggestion
   dpgContext.rlcOptions = transformRLCOptions(emitterOptions, dpgContext);
```
Is this not working ?

Reply:
Just realized options is being used somewhere else ?

Reply:
wonder if there's any place that will modify the options causing them to be inconsistent.

Reply:
I prefer to refactor this code in my next pr.